kind.yaml

```diff
@@ -19,16 +19,17 @@ pipeline:
     with:
       repository: https://github.com/kubernetes-sigs/kind
       tag: v${{package.version}}
-      expected-commit: 2b248e7df157d4f1a44ecea114be3d58c9232930
-
-  - uses: go/bump
-    with:
-      deps: golang.org/x/sys@v0.0.0-20220412211240-33da011f77ad
+      expected-commit: 0296c52b38a6bb5ccdcbcf4e57f0ba79415e6ff2
 
   - runs: |
-      make build
-      mkdir -p ${{targets.destdir}}/usr/bin/
-      cp ./bin/kind ${{targets.destdir}}/usr/bin/kind
+      sed -i 's/golang.org\/x\/sys v0.0.0-20210630005230-0f9fa26af87c/golang.org\/x\/sys v0.0.0-20220412211240-33da011f77ad/' go.mod
```

Comment:
Can we do this using gobump somehow? That way automation will be able to remove this automatically when upstream changes.

Reply:
i try first with go/bump but didnt solve the problem, so I found this solution.

Reply:
lets try again